PublicSDKAPITester/RCPurchasesErrorUtilsAPI.m

```diff
@@ -0,0 +1,37 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  RCPurchasesErrorUtilsAPI.m
+//
+//  Created by César de la Vega on 7/21/21.
+
+@import Purchases;
+@import PurchasesCoreSwift;
+
+#import "RCPurchasesErrorUtilsAPI.h"
+
+@implementation RCPurchasesErrorUtilsAPI
```

Comment:
why did we ever have this public? it doesn't seem particularly useful to devs

Reply:
Yeah idk. Maybe for testing purposes? So they can create errors like ours. But I think it was just an oversight